### PR TITLE
Table version selection dialog

### PR DESCRIPTION
**Pull request by Nikhil-lab**
1. Organs version mismatch fixed
2.  The list is sorted in alphabetical order,
3. It should have an "All organs" like it used to have with the same v1.0 or 1.1 options.-- Done

### CODE REVIEW

**Comment by vutukuriajay2241** [comment]
- After creating a sheet-config.json, Directly import data from sheet-config.json to config.ts.
- Moving sheet-config.json into assets Folder.
- Import the Json file at the runtime not at the compile time.
- Please try to optimize the code
- At some parts, code is duplicating try to remove that.
- Fix the code smells which are thrown by Sonar cloud

File: projects/v2/src/app/json-typings.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+declare module '*.json' {
```
Comment (bherr2) [inline projects/v2/src/app/json-typings.d.ts:1]: No. The JSON data should be loaded dynamically at runtime via fetch/httpclient from assets.

Comment (Nikhil-lab) [reply projects/v2/src/app/json-typings.d.ts:1]: ok got it

**Comment by bherr2** [comment]
Is this superceded by your work in #308 ?

**Comment by Nikhil-lab** [comment]
yes Bruce

**Comment by bherr2** [comment]
Closing in favor of #308

**Pull request closed without merging**